common/persistence/elasticsearch/esProcessor_test.go

```diff
@@ -142,7 +140,7 @@ func (s *esProcessorSuite) TestAdd() {
 	s.Equal(0, s.esProcessor.mapToAckChan.Len())
 
 	s.mockBulkProcessor.EXPECT().Add(request).Times(1)
-	s.mockMetricClient.On("StartTimer", testScope, testMetric).Return(testStopWatch).Once()
+	s.mockMetricClient.EXPECT().StartTimer(testScope, testMetric).Return(testStopWatch)
```

Comment:
`Times(1)` seems to be default in `gomock`. Can you remove it from everywhere with next PR?

Reply:
Yes. But I support this to be a separate PR.